Makefile

```diff
@@ -50,16 +51,19 @@ generate: packr2
 	$(GO) mod tidy
 	$(GO) generate ./...
 
-HAS_PACKR2 := $(shell command -v packr2)
-HAS_GOBIN_IN_PATH := $(shell re='(:|^)$(CLIENT_GOPATH)/bin/?(:|$$)'; if [[ "$${PATH}" =~ $${re} ]];then echo $${GOPATH}/bin;fi)
+HAS_PACKR2 := $(shell command -v packr3 && echo "OK")
```

Comment:
`packr3` here is a typo, right?

Reply:
What is the utility of the `&& echo "OK"` portion of this update?

Reply:
This is a left over of me testing the negative case for this command. The `&& echo "OK` makes the command work for me on GitBash. Otherwise, it was always returning negative.

Reply:
Fixed.